### ISSUE TITLE
export 'NgxDropzonePreviewComponent' was not found in 'ngx-dropzone'

### ISSUE THREAD

**Issue by Tazaf** [peterfreeman/ngx-dropzone#37]
Hi!

I'm trying to implement a custom file preview following the ["How to create a custom preview component" wiki page](https://github.com/peterfreeman/ngx-dropzone/wiki/How-to-create-a-custom-preview-component%3F).

But I've got an issue with this line...
```typescript
import { NgxDropzonePreviewComponent } from 'ngx-dropzone';
```
...which causes the following compilation error:
```
Failed to compile.

./src/app/routes/product/pdf-dropzone-preview/pdf-dropzone-preview.component.ts 6:108-135
"export 'NgxDropzonePreviewComponent' was not found in 'ngx-dropzone'
```

Seems like the `NgxDropzonePreviewComponent` is not exported by the `ngx-dropzone` module.

VS Code auto detects an `NgxDropzonePreviewComponent` here:
```typescript
import { NgxDropzonePreviewComponent } from 'ngx-dropzone/lib/ngx-dropzone-preview/ngx-dropzone-preview.component';
```
Although the Angular compiler is not very happy with this:
```
Failed to compile.

./src/app/routes/product/pdf-dropzone-preview/pdf-dropzone-preview.component.ts
Module not found: Error: Can't resolve 'ngx-dropzone/lib/ngx-dropzone-preview/ngx-dropzone-preview.component' in '/mnt/c/Square/media-f/mediashop/mediashop-front/src/app/routes/product/pdf-dropzone-preview'
```
**Note:** I'm using `ngx-dropzone@2.1.0`

Is this an issue with the wiki page, the module or me 😄 ?

Thanks in advance!

**Comment by Tazaf** [comment]
I have found a workaround by importing this:
```
import { ɵc as NgxDropzonePreviewComponent } from 'ngx-dropzone';
```

But this seems very hacky to me :)

**Comment by peterfreeman** [comment]
Hey @Tazaf,

thanks for your feedback. Your are right, I forgot to export the `NgxDropzonePreviewComponent` in the `public_api.ts` file. I just released `v2.1.1` which includes a fix, check it out!

May I ask what type of preview component you're trying to implement? Maybe it's useful for others too and we could include it in the library?

Happy coding! 🤓

**Comment by Tazaf** [comment]
Hi @peterfreeman,

Thanks for the update, I'll check it out for sure pretty soon 😉 

As for the preview compoenent, I was trying to build a PDF preview component, mixing your lib and [`ng2-pdf-preview`](https://www.npmjs.com/package/ng2-pdf-viewer).

I manage to display the first page of the PDF as a preview in the dropzone, but there was some issue regarding the size of said preview (way too big compared to the dropzone itself). I did not investigate further though, as I've taken another path more fitting the needs of my app, but it would def' be feasable.
